tools/sigmut/sigmut_macros.xml

```diff
@@ -0,0 +1,23 @@
+<macros>
+      
+    <token name="@VERSION@">1.0</token>
```

Comment:
... and at this stage, should not be 1.x but 0.9.0 to indicate that it is a beta version

Reply:
Yes, that's OK.